docs/reference/ml/df-analytics/apis/put-inference.asciidoc

```diff
@@ -382,6 +382,10 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=inference-config-results-field]
 `top_classes_results_field`::::
 (Optional, string)
 include::{docdir}/ml/ml-shared.asciidoc[tag=inference-config-classification-top-classes-results-field]
+
+`prediction_field_type`::::
```

Comment:
> prediction_field_type

Not a big deal, but these properties are typically sorted alphabetically, and this seems to fall out of that sort order.